test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -485,32 +487,34 @@ export default class TabularView extends Component {
         let found = false;
 
         data.forEach(function (testResultsObject) {
-            platform = testResultsObject.buildName
-            let aggregateIndex = 0;
-            for(; aggregateIndex < testResultsObject.aggregateInfo.length; aggregateIndex++) {
-                for (const metric in testResultsObject.aggregateInfo[aggregateIndex].metrics) {
-                    found = false;
-                    benchmarkNVM = testResultsObject.aggregateInfo[aggregateIndex].benchmarkName + ',' + testResultsObject.aggregateInfo[aggregateIndex].benchmarkVariant + "," + testResultsObject.aggregateInfo[aggregateIndex].metrics[metric].name;
+            const buildInfo = getInfoFromBuildName(testResultsObject.buildName);
+            if (buildInfo){
+                platform = buildInfo.platform;
+                let aggregateIndex = 0;
+                for(; aggregateIndex < testResultsObject.aggregateInfo.length; aggregateIndex++) {
```

Comment:
remove line 493. Change line 494:

for(let aggregateIndex = 0; aggregateIndex < testResultsObject.aggregateInfo.length; aggregateIndex++) {